lib/rust/parser/src/macros.rs

```diff
@@ -0,0 +1,40 @@
+//! The macro system for the Enso parser.
```

Comment:
We are not using plural names in modulee naes. This sghould be named macro (until its reservedword, is it?)

Reply:
It is a reserved word unfortunately. That's why I used `macros`.